druntime/src/core/thread/fiber.d

```diff
@@ -1467,6 +1450,32 @@ private:
              */
             pstack += int.sizeof * 1;
         }
+        else version (AsmLoongArch64_Posix)
```

Comment:
please keep this in the same spot (after `AsmMIPS_O32_Posix`) so that its is easier to review